hikari/impl/stateless_cache.py

```diff
@@ -72,21 +72,21 @@ def clear_emojis(self) -> cache.CacheView[snowflakes.Snowflake, emojis.KnownCust
         raise self._no_cache()
 
     def clear_emojis_for_guild(
-        self, guild_id: snowflakes.Snowflake, /
+        self, guild_id: snowflakes.Snowflakeish, /
```

Comment:
Could we allow objects too? ( Just like we do in the REST)

```suggestion
        self, guild_id: snowflakes.SnowflakeishOr[guilds.Guild], /
```

Then we can rename `guild_id` to just `guild`. This will remove a lot of complexity on calls outside the cache (specially for users) as they wont have to check what to pass, they can pass any object

Reply:
Could objects not have a get self method as a random idea?

Reply:
Also the arg name doesn't really matter here too much since they're positional only anyway

Reply:
Arg name is just for consistency

Reply:
might save that change for another pr just cause bleh